apps/omg_watcher_rpc/lib/web/plugs/supported_watcher_modes.ex

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherRPC.Web.Plugs.SupportedWatcherModes do
+  @moduledoc """
+  A plug that serves the endpoints in scope only when the running watcher's mode matches
```

Comment:
I find it hard to get through this sentence. How about "Ensures that unsupported endpoints return an appropriate error, depending on watcher's mode"?

Reply:
1aa0f3d